projects/media-viewer/src/lib/annotations/annotation-set/annotation/comment/comment.component.html

```diff
@@ -1,4 +1,4 @@
-  <form (click)="onCommentClick()" class="aui-comment"
+  <form #form (click)="onCommentClick()" class="aui-comment"
```

Comment:
what is the #form for?

Reply:
to grab native element and its parent to eventually get clientWidth